python-sdk/tests/test_prompt_formatter.py

```diff
@@ -1,26 +1,26 @@
 import pytest
```

Comment:
Do we want to use liquid like we do in JS?

Reply:
yup. forgot to do a lot of the python stuff